### PR TITLE
@W-11852162@ Add commerce:scratchorg:create

### PR DESCRIPTION
**Pull request by unandyala**
The code is basically copy of `commerce:scratchorg:create` from commerce-on-lightning-orchestration repo with some cleaning.

Also updated the version to 57.0

Verified that the below commands successfully create the org
`./bin/run commerce:scratchorg:create -u demo_u21@1commerce.com -a u21 -t both -v ceo@devhub2.com --apiversion=57.0`

`./bin/run commerce:scratchorg:create --targetusername demo_u23@1commerce.com --alias u23 --type both --targetdevhubusername ceo@devhub2.com --apiversion=57.0
`

### CODE REVIEW

File: src/commands/commerce/scratchorg/create.ts [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    private devHubDir: string;
+
+    public async run(): Promise<AnyJson> {
+        // copy the config files
+        FILE_COPY_ARGS.forEach((v) => modifyArgFlag(v.args, v.value, this.argv));
+        await FilesCopy.run(addAllowedArgs(this.argv, FilesCopy), this.config);
+        this.statusManager = new StatusFileManager(this.flags.devhubUsername, this.flags['scratch-org-admin-username']);
+        this.ux.log(msgs.getMessage('create.usingScratchOrgAdmin', [this.flags['scratch-org-admin-username']]));
+        if (!getOrgInfo(this.flags['hub-org-admin-username']))
+            // TODO add this as a require, ie requires devhub
```
[the remark this answers is not in the document]
Comment (unandyala) [reply src/commands/commerce/scratchorg/create.ts:67]: There is a helper class that does all prereqs. This todo says to move the org validation to that helper. This is an existing one

**Review by tarcang — DISMISSED** [review]

**Review by tarcang — DISMISSED** [review]

**Review by joeluong-sfcc — DISMISSED** [review]

**Review by tarcang — DISMISSED** [review]

**Review by joeluong-sfcc — DISMISSED** [review]

**Review by tarcang — DISMISSED** [review]

**Review by joeluong-sfcc — DISMISSED** [review]
Is there a specific reason you reverted the API version from `57.0` back to `52.0`?

**Comment by unandyala** [comment]
> Is there a specific reason you reverted the API version from `57.0` back to `52.0`?

yes.. the version is not passed correctly inside the code and this is breaking the steam env. We have a separate story to fix the version. As part of that story, we will update the version to 57.0

**Comment by unandyala** [comment]
closing this as the commits are not signed

**Pull request closed without merging**